### PR TITLE
renamed from/to to source/target

### PR DESCRIPTION
**Pull request by yudilevi**

### CODE REVIEW

File: solidity/contracts/BancorNetwork.sol [hunk 1]
```diff
@@ -56,12 +56,12 @@ contract BancorNetwork is IBancorNetwork, TokenHolder, ContractRegistryClient, F
...
       * @dev triggered when a conversion between two tokens occurs
       * 
-      * @param _smartToken      smart token governed by the converter
-      * @param _fromToken       ERC20 token converted from
-      * @param _toToken         ERC20 token converted to
-      * @param _fromAmount      amount converted, in fromToken
-      * @param _toAmount        amount returned, minus conversion fee
-      * @param _trader          wallet that initiated the trade
+      * @param _smartToken  smart token governed by the converter
+      * @param _fromToken   source ERC20 token
```
Comment (barakman) [inline solidity/contracts/BancorNetwork.sol:60]: @yudilevi : Did you miss these two, or would it break current API?

File: solidity/contracts/converter/BancorConverter.sol [hunk 2]
```diff
@@ -56,10 +56,10 @@ contract BancorConverter is IBancorConverter, TokenHandler, SmartTokenController
     /**
       * @dev triggered when a conversion between two tokens occurs
       * 
-      * @param _fromToken       ERC20 token converted from
-      * @param _toToken         ERC20 token converted to
+      * @param _fromToken       source ERC20 token
```
Comment (barakman) [inline solidity/contracts/converter/BancorConverter.sol:59]: @yudilevi : Did you miss these two, or would it break current API?

File: solidity/contracts/BancorNetwork.sol (hunk 1 above)
Comment (yudilevi) [reply solidity/contracts/BancorNetwork.sol:60]: yeah, will break the API

File: solidity/contracts/converter/BancorConverter.sol (hunk 2 above)
Comment (yudilevi) [reply solidity/contracts/converter/BancorConverter.sol:59]: same - will break the API

**Pull request merged**